client/src/components/Card.js

```diff
@@ -33,7 +21,7 @@ export default function Card({ imgsrc, eventDatas, eventID, onDelete }) {
     <CardContainer imgsrc={imgsrc}>
       <CustomLink to={`/eventDetails/?ID=${eventID}`}>
         <DarkFilter />
-        <Headline>{eventDatas.title ? eventDatas.title : "Event"}</Headline>
+        <Headline>{eventDatas.title || "Event"}</Headline>
```

Comment:
Not related to this PR, but `datas` is not a valid word.
Maybe `event` or `eventData` is a better name?